src/Connection.php

```diff
@@ -166,6 +167,12 @@ class Connection
 
     private SchemaManagerFactory $schemaManagerFactory;
 
+    private static bool $isChecking = false;
+
+    private static ?int $lastCheckedAt = null;
```

Comment:
do we really need null? what about defaulting to `0`?

Reply:
Yes, indeed, I think that works too :)